database_cleanup/model/purge_wizard.py

```diff
@@ -1,78 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
that's what I did: https://github.com/OCA/server-tools/commits/d37bfe7be687d222a212d43f77a064d6ee8e6a42/database_cleanup/model/purge_wizard.py - github has trouble following file renames sometimes